torch/autograd/grad_mode.py

```diff
@@ -254,6 +255,35 @@ def __exit__(self, *args) -> None:
     def clone(self):
         return self.__class__(self.mode)
 
+class _unsafe_preserve_version_counter(_DecoratorContextManager):
+    r"""Context-manager that unsafely tells autograd to ignore mutations to a tensor.
```

Comment:
Could you make this more scarry please?
```suggestion
    r"""DO NOT USE THIS UNLESS YOU KNOW EXACTLY WHAT YOU'RE DOING!
    This context manager can lead to arbitrary silent-correctness issues in any other part of your code (even the ones not touched directly by the context manager)!
    
    Context-manager that unsafely tells autograd to ignore mutations to a tensor.
```

Reply:
absolutely 😛